---
title: A contrastive learning approach to integrate spatial transcriptomics and histological
  images
date: '2024-05-06'
linkTitle: https://pubmed.ncbi.nlm.nih.gov/38707535/?utm_source=curl&utm_medium=rss&utm_campaign=pubmed-2&utm_content=1x5bM_TNL8gjogAcnslpo2s2PbDe-61JVM2h9yowOYSiZ7Dkrt&fc=20220919211934&ff=20240507180352&v=2.18.0.post9+e462414
source: (graph neural network) AND (gene)
description: The rapid growth of spatially resolved transcriptomics technology provides
  new perspectives on spatial tissue architecture. Deep learning has been widely applied
  to derive useful representations for spatial transcriptome analysis. However, effectively
  integrating spatial multi-modal data remains challenging. Here, we present ConGcR,
  a contrastive learning-based model for integrating gene expression, spatial location,
  and tissue morphology for data representation and spatial tissue ...
disable_comments: true
---
The rapid growth of spatially resolved transcriptomics technology provides new perspectives on spatial tissue architecture. Deep learning has been widely applied to derive useful representations for spatial transcriptome analysis. However, effectively integrating spatial multi-modal data remains challenging. Here, we present ConGcR, a contrastive learning-based model for integrating gene expression, spatial location, and tissue morphology for data representation and spatial tissue ...